docs/howto/craft-for-platforms/example-java-app.rst

```diff
@@ -0,0 +1,127 @@
+.. _example-java-app:
+
+Example Java app
+================
+
+This how-to guide covers the steps, decisions, and implementation details that
+are unique when crafting a Java-based snap. We'll work through the aspects
+unique to Java apps by examining an existing recipe.
+
+The process of developing a snap for a Java app builds on top of JRE, making it
+possible to adapt and integrate an app's existing build tooling and
+dependencies into the crafting process.
+
+
+Example recipe for Cal
```

Comment:
Just for me!

Reply:
https://en.wikipedia.org/wiki/(Everything_I_Do)_I_Do_It_for_You